joschahenningsen/TUM-Live
# adding shortcuts on the video player buttons controltext

### Motivation and Context



fixes #1289

### Description

- [x] shortcuts are added to video players buttons. before: play -> after play (k)] 
- [x] keyboard shortcuts should be added to the menu under keyboard icon

### Steps for Testing


1. Log in
2. Navigate to a Video
3. Check the text appears when the mouse is on one of the player buttons
4. Open menu at the home page and click keyboard icon

### Screenshots

It used to say only play, not the keyboard shortcuts are also displayed.
![New_tooltips_when_mouse_on_video_player_button](https://github.com/TUM-Dev/gocast/assets/70471039/b6f3d112-16d4-4c53-9e1a-d704e49cd7a0)
Keyboard shortcuts in menu:
![Screenshot 2024-01-28 at 23 25 12](https://github.com/TUM-Dev/gocast/assets/70471039/54fd3c00-f9a7-4ad7-9088-d1b3689cbf32)
Keyboard shortcuts popup:
![Screenshot 2024-01-28 at 23 24 34](https://github.com/TUM-Dev/gocast/assets/70471039/83f9dec9-5589-4b82-abfb-4925a15bff96)
